[2024-09-23T01:02:24.603+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-23T01:02:24.668+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mercadolivre_scraper_to_snowflake_dag.insert_into_snowflake manual__2024-09-23T00:57:08.862939+00:00 [queued]>
[2024-09-23T01:02:24.705+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mercadolivre_scraper_to_snowflake_dag.insert_into_snowflake manual__2024-09-23T00:57:08.862939+00:00 [queued]>
[2024-09-23T01:02:24.709+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-23T01:02:24.767+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): insert_into_snowflake> on 2024-09-23 00:57:08.862939+00:00
[2024-09-23T01:02:24.796+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=592) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-23T01:02:24.805+0000] {standard_task_runner.py:72} INFO - Started process 595 to run task
[2024-09-23T01:02:24.810+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mercadolivre_scraper_to_snowflake_dag', 'insert_into_snowflake', 'manual__2024-09-23T00:57:08.862939+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/mercado_livre.py', '--cfg-path', '/tmp/tmppljyop1w']
[2024-09-23T01:02:24.814+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask insert_into_snowflake
[2024-09-23T01:02:25.046+0000] {task_command.py:467} INFO - Running <TaskInstance: mercadolivre_scraper_to_snowflake_dag.insert_into_snowflake manual__2024-09-23T00:57:08.862939+00:00 [running]> on host 6a22fc4abd55
[2024-09-23T01:02:25.382+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mercadolivre_scraper_to_snowflake_dag' AIRFLOW_CTX_TASK_ID='insert_into_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2024-09-23T00:57:08.862939+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-23T00:57:08.862939+00:00'
[2024-09-23T01:02:25.390+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-23T01:02:25.392+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-23T01:02:25.393+0000] {logging_mixin.py:190} INFO - Current task name:insert_into_snowflake state:running start_date:2024-09-23 01:02:24.674318+00:00
[2024-09-23T01:02:25.395+0000] {logging_mixin.py:190} INFO - Dag name:mercadolivre_scraper_to_snowflake_dag and current dag run status:running
[2024-09-23T01:02:25.398+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-23T01:02:25.446+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.2, Python Version: 3.12.6, Platform: Linux-6.8.0-1015-gcp-x86_64-with-glibc2.36
[2024-09-23T01:02:25.447+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2024-09-23T01:02:25.448+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-09-23T01:02:26.036+0000] {mercado_livre.py:65} ERROR - Error: 'str' object has no attribute 'product_link'
[2024-09-23T01:02:26.355+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2024-09-23T01:02:26.356+0000] {connection.py:788} INFO - closed
[2024-09-23T01:02:26.401+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2024-09-23T01:02:26.435+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-23T01:02:26.450+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-23T01:02:26.450+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mercadolivre_scraper_to_snowflake_dag, task_id=insert_into_snowflake, run_id=manual__2024-09-23T00:57:08.862939+00:00, execution_date=20240923T005708, start_date=20240923T010224, end_date=20240923T010226
[2024-09-23T01:02:26.464+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-23T01:02:26.465+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-23T01:02:26.465+0000] {logging_mixin.py:190} INFO - Dag name:mercadolivre_scraper_to_snowflake_dag queued_at:2024-09-23 00:57:08.882462+00:00
[2024-09-23T01:02:26.466+0000] {logging_mixin.py:190} INFO - Task hostname:6a22fc4abd55 operator:PythonOperator
[2024-09-23T01:02:26.498+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-23T01:02:26.521+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-23T01:02:26.523+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
